.github/workflows/basic.yml

```diff
@@ -208,11 +210,28 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Get CMake for lukka/run-vcpkg
+      uses: lukka/get-cmake@latest
+      with:
+        cmakeVersion: 3.22.6
+
+    - name: Initialize vcpkg
+      uses: lukka/run-vcpkg@v11.5
+      with:
+        vcpkgGitCommitId: 3dd44b931481d7a8e9ba412621fa810232b66289
+        vcpkgDirectory: ${{ github.workspace }}/build/vcpkg
+        vcpkgConfigurationJsonGlob: '**/vcpkg-configuration.json'
+
+    - name: Install dependencies
```

Comment:
can this be run as part of `lukka/run-vcpkg` action...? it seems so, based on their README page.

if we could use only 1 action instead of current 3 actions, it would be really nice 😉

Reply:
I tried and didn't succeed. Someone can play with it later.